pages/index.js

```diff
@@ -154,6 +185,7 @@ const Home = ({ latestStories }) => {
     await axios.get('/api/fetch?url=https://recommender.thedp.com/covid').then(resp => {
       const { data: { results } } = resp
       setCumulativeCase(results[0]["Positive_Cases_Cumulative"][results[0]["Positive_Cases_Cumulative"].length - 1])
+      setCumulativeTest(results[0]["Tests_Done_Cumulative"][results[0]["Tests_Done_Cumulative"].length - 1])
```

Comment:
prefer rewriting this to destructure out the first elem of results using array destructuring and also destructuring out Positive_Cases_Cum and Tests_Done_Cum in an object destructure. Will add a few lines but will improve readability imo